swarm/storage/localstore.go

```diff
@@ -28,8 +28,9 @@ import (
 
 type LocalStoreParams struct {
 	*StoreParams
-	ChunkDbPath string
-	Validators  []ChunkValidator `toml:"-"`
+	ChunkDbPath    string
+	Validators     []ChunkValidator `toml:"-"`
+	SkipValidation bool
```

Comment:
Please add a comment that this is used only for tests, and also elaborate on the kind of tests this is used for.